build/__variables.sh

```diff
@@ -26,6 +26,7 @@ declare -r ORYXTESTS_SLIM_BUILDIMAGE_DOCKERFILE="$REPO_DIR/tests/images/build/sl
 declare -r RUNTIME_IMAGES_SRC_DIR="$REPO_DIR/images/runtime"
 declare -r BUILD_IMAGES_CLI_DOCKERFILE="$REPO_DIR/images/build/cli.Dockerfile"
 declare -r BUILD_IMAGES_AZ_FUNCS_JAMSTACK_DOCKERFILE="$REPO_DIR/images/build/AzureFunctions.JamStack.Dockerfile"
+declare -r BUILD_IMAGES_GITHUB_ACTIONS_DOCKERFILE="$REPO_DIR/images/build/GitHubActions.Dockerfile"
```

Comment:
nit: I may be thinking too much … but I feel its more like base image for all the future githubaction images .. so can we name it accordingly?

Reply:
The thing is this image is strongly tied to the layers of the image that the GitHub agent machines have so we might not want it to be the base image. AppService could use latest version of buildpack-deps having the security patches and need not be strongly tied to the one which GitHub action uses.